docs/framework/develop-web-apps-with-aspnet.md

```diff
@@ -8,13 +8,15 @@ helpviewer_keywords:
 ms.assetid: b7861df0-690a-4a58-bd12-f9d0123e40df
 ---
 # Developing Web Applications with ASP.NET
-ASP.NET is a .NET Framework technology for creating web apps. The following links are provided for your convenience:  
+ASP.NET is a .NET Framework technology for creating web apps. For more information on ASP.NET, see:
 
-- [ASP.NET Core Documentation](/aspnet/core/)   
 - [ASP.NET documentation](/aspnet/overview)   
 - [ASP.NET MVC](https://go.microsoft.com/fwlink/p/?LinkID=227227)   
 - [ASP.NET Web Pages](https://go.microsoft.com/fwlink/p/?LinkId=251040)   
 - [ASP.NET Web API](https://go.microsoft.com/fwlink/p/?LinkId=251041)  
+- [Create an ASP.NET Framework web app in Azure](/azure/app-service/app-service-web-get-started-dotnet-framework)
+- [ASP.NET Core Documentation](/aspnet/core/)   
```

Comment:
Since this is explicitly a list of ASP.NET articles, the ASP.NET Core link strikes me as being out of place. I think that it merits a separate introductory paragraph ("You can also develop web apps with ASP.NET Core, a related technology that... For more information, see:") with links to the ASP.NET Core documentation and a getting started article.